src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,45 @@
+.navigation {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style-type: none;
+    align-items: flex-end;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), vertical alignment should be consistent throughout the project. It's recommended to use `center` instead of `flex-end` to align items vertically, unless a specific design requirement dictates otherwise.